core/src/subgraph/instance_manager.rs

```diff
@@ -806,6 +799,24 @@ where
     }
 }
 
+async fn delete_subgraph_firehose_cursor(logger: &Logger, store: &dyn WritableStore) {
+    debug!(logger, "Deleting any existing Firehose cursor");
+    let mut backoff = ExponentialBackoff::new(30 * SECOND, *SUBGRAPH_ERROR_RETRY_CEIL_SECS);
```

Comment:
I agree, in the `FirehoseBlockIngestor` there is a bunch of those also so it would be a good thing to have a re-usable method. Again, would like to postpone that to another PR.